src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx

```diff
@@ -383,15 +384,24 @@ export default function TransactionConfirmationModal({
   operationType, // mod
```

Comment:
[`| undefined`](https://github.com/cowprotocol/cowswap/blob/10b2abe6c59d9a2a0a3abc190ff7845ff1bd0ab1/src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx#L275) is redundant here!